src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+
+    private String name = "Circle";
+
+    private String color;
+    private double radius;
+
+    public Circle(double radius, String color) {
+        this.radius = radius;
+        this.color = color;
```

Comment:
Creating a circle, don't pass expressions in the constructor. Create separate variables for 'radius' and 'color' and pass them on for better code readability.